### PR TITLE
fix(curriculum): Remove unnecessary assert message argument from English Coding Interview Prep challenges - 02

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to issue #36400

This PR removes assert message arguments from `testString` for the following sub section of the `Coding Interview Prep` curriculum section:
* Rosetta Code

### CODE REVIEW

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/align-columns.english.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ or$center$justified$within$its$column.
 ```yml
 tests:
   - text: <code>formatText</code> is a function.
-    testString: assert(typeof formatText === 'function', '<code>formatText</code> is a function.');
+    testString: assert(typeof formatText === 'function');
   - text: '<code>formatText</code> with the above input and "right" justification should produce the following: '
     testString: 'assert.strictEqual(formatText(testInput, ''right''), rightAligned, ''<code>formatText</code> with the above input and "right" justification should produce the following: '');'
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/align-columns.english.md]: There are a few message arguments in `testString` that needs to be removed in this file.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/deal-cards-for-freecell.english.md [hunk 2]
```diff
@@ -76,11 +76,11 @@ Deals can also be checked against <a href="https://freecellgamesolutions.com/" t
...
-    testString: assert(typeof dealFreeCell === 'function', '<code>dealFreeCell</code> is a function.');
+    testString: assert(typeof dealFreeCell === 'function');
   - text: <code>dealFreeCell(seed)</code> should return an object.
-    testString: assert(typeof dealFreeCell(1) === 'object', '<code>dealFreeCell(seed)</code> should return an object.');
+    testString: assert(typeof dealFreeCell(1) === 'object');
   - text: <code>dealFreeCell(seed)</code> should return an array of length 7.
-    testString: assert(dealFreeCell(1).length === 7, '<code>dealFreeCell(seed)</code> should return an array of length 7.');
+    testString: assert(dealFreeCell(1).length === 7);
   - text: "<code>dealFreeCell(1)</code> should return an array identical to example \"Game #1\""
     testString: "assert.deepEqual(dealFreeCell(1), game1, '<code>dealFreeCell(1)</code> should return an array identical to example \"Game #1\"');"
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/deal-cards-for-freecell.english.md]: 2 message arguments need to be removed in this file.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/deepcopy.english.md [hunk 3]
```diff
@@ -27,15 +27,15 @@ This task will not test for:
 ```yml
 tests:
   - text: <code>deepcopy</code> should be a function.
-    testString: assert(typeof deepcopy === 'function', '<code>deepcopy</code> should be a function.');
+    testString: assert(typeof deepcopy === 'function');
   - text: '<code>deepcopy({test: "test"})</code> should return an object.'
     testString: 'assert(typeof deepcopy(obj1) === ''object'', ''<code>deepcopy({test: "test"})</code> should return an object.'');'
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/deepcopy.english.md]: This one here.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/factorial.english.md [hunk 4]
```diff
@@ -30,9 +30,9 @@ For example:
 ```yml
 tests:
   - text: <code>factorial</code> is a function.
-    testString: assert(typeof factorial === 'function', '<code>factorial</code> is a function.');
+    testString: assert(typeof factorial === 'function');
   - text: <code>factorial(2)</code> should return a number.
-    testString: assert(typeof factorial(2) === 'number', '<code>factorial(2)</code> should return a number.');
+    testString: assert(typeof factorial(2) === 'number');
   - text: <code>factorial(3)</code> should return 6.
     testString: assert.equal(factorial(3),results[0],"<code>factorial(3)</code> should return 6.");
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/factorial.english.md]: Remove 3 arguments in this file.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/fibonacci-sequence.english.md [hunk 5]
```diff
@@ -24,9 +24,9 @@ Hence, the series is: 0, 1, 1, 2, 3, 5, 8, 13...
 ```yml
 tests:
   - text: <code>fibonacci</code> is a function.
-    testString: assert(typeof fibonacci === 'function', '<code>fibonacci</code> is a function.');
+    testString: assert(typeof fibonacci === 'function');
   - text: <code>fibonacci(2)</code> should return a number.
-    testString: assert(typeof fibonacci(2) == 'number', '<code>fibonacci(2)</code> should return a number.');
+    testString: assert(typeof fibonacci(2) == 'number');
   - text: <code>fibonacci(3)</code> should return 1.
     testString: assert.equal(fibonacci(3),1,"<code>fibonacci(3)</code> should return 1.");
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/fibonacci-sequence.english.md]: 3 arguments here too.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/general-fizzbuzz.english.md [hunk 6]
```diff
@@ -24,15 +24,15 @@ The second parameter is the number for which the function should return a string
...
+    testString: assert.equal(genFizzBuzz([[3, "Fizz"],[5, "Buzz"]], 6), "Fizz");
   - text: <code>genFizzBuzz([[3, "Fizz"],[5, "Buzz"]], 10)</code> should return <code>"Buzz"</code>.
-    testString: assert.equal(genFizzBuzz([[3, "Fizz"],[5, "Buzz"]], 10), "Buzz",'<code>genFizzBuzz([[3, "Fizz"],[5, "Buzz"]], 10)</code> should return <code>"Buzz"</code>.');
+    testString: assert.equal(genFizzBuzz([[3, "Fizz"],[5, "Buzz"]], 10), "Buzz");
   - text: <code>genFizzBuzz([[3, "Buzz"],[5, "Fizz"]], 12)</code> should return <code>"Buzz"</code>.
-    testString: assert.equal(genFizzBuzz([[3, "Buzz"],[5, "Fizz"]], 12), "Buzz",'<code>genFizzBuzz([[3, "Buzz"],[5, "Fizz"]], 12)</code> should return <code>"Buzz"</code>.');
+    testString: assert.equal(genFizzBuzz([[3, "Buzz"],[5, "Fizz"]], 12), "Buzz");
   - text: <code>genFizzBuzz([[3, "Buzz"],[5, "Fizz"]], 13)</code> should return <code>"13"</code>.
     testString: assert.equal(genFizzBuzz([[3, "Buzz"],[5, "Fizz"]], 13), '13','<code>genFizzBuzz([[3, "Buzz"],[5, "Fizz"]], 13)</code> should return <code>"13"</code>.');
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/general-fizzbuzz.english.md]: 4 arguments in this file.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/harshad-or-niven-series.english.md [hunk 7]
```diff
@@ -23,7 +23,7 @@ Use it to list the first twenty members of the sequence and list the first Harsh
 ```yml
 tests:
   - text: <code>isHarshadOrNiven</code> is a function.
-    testString: assert(typeof isHarshadOrNiven === 'function', '<code>isHarshadOrNiven</code> is a function.');
+    testString: assert(typeof isHarshadOrNiven === 'function');
   - text: '<code>isHarshadOrNiven()</code> should return <code>{"firstTwenty": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 18, 20, 21, 24, 27, 30, 36, 40, 42],"firstOver1000": 1002}</code>'
     testString: 'assert.deepEqual(isHarshadOrNiven(), res, ''<code>isHarshadOrNiven()</code> should return <code>{"firstTwenty": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 18, 20, 21, 24, 27, 30, 36, 40, 42],"firstOver1000": 1002}</code>'');'
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/harshad-or-niven-series.english.md]: This one.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/hash-from-two-arrays.english.md [hunk 8]
```diff
@@ -24,7 +24,7 @@ Using two Arrays of equal length, create a Hash object where the elements from o
 ```yml
 tests:
   - text: <code>arrToObj</code> is a function.
-    testString: assert(typeof arrToObj === 'function', '<code>arrToObj</code> is a function.');
+    testString: assert(typeof arrToObj === 'function');
   - text: '<code>arrToObj([1, 2, 3, 4, 5], ["a", "b", "c", "d", "e"])</code> should return <code>{ 1: "a", 2: "b", 3: "c", 4: "d", 5: "e" }</code>'
     testString: 'assert.deepEqual(arrToObj(...testCases[0]), res[0], ''<code>arrToObj([1, 2, 3, 4, 5], ["a", "b", "c", "d", "e"])</code> should return <code>{ 1: "a", 2: "b", 3: "c", 4: "d", 5: "e" }</code>'');'
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/hash-from-two-arrays.english.md]: Fix this whole file.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/hash-join.english.md [hunk 9]
```diff
@@ -173,7 +173,7 @@ The order of the rows in the output table is not significant.
...
     testString: 'assert.deepEqual(hashJoin(hash1, hash2), res, ''<code>hashJoin([{ age: 27, name: "Jonah" }, { age: 18, name: "Alan" }, { age: 28, name: "Glory" }, { age: 18, name: "Popeye" }, { age: 28, name: "Alan" }], [{ character: "Jonah", nemesis: "Whales" }, { character: "Jonah", nemesis: "Spiders" }, { character: "Alan", nemesis: "Ghosts" }, { character:"Alan", nemesis: "Zombies" }, { character: "Glory", nemesis: "Buffy" }, { character: "Bob", nemesis: "foo" }])</code> should return <code>[{"A_age": 27,"A_name": "Jonah", "B_character": "Jonah", "B_nemesis": "Whales"}, {"A_age": 27,"A_name": "Jonah", "B_character": "Jonah", "B_nemesis": "Spiders"}, {"A_age": 18,"A_name": "Alan", "B_character": "Alan", "B_nemesis": "Ghosts"}, {"A_age": 18,"A_name": "Alan", "B_character": "Alan", "B_nemesis": "Zombies"}, {"A_age": 28,"A_name": "Glory", "B_character": "Glory", "B_nemesis": "Buffy"}, {"A_age": 28,"A_name": "Alan", "B_character": "Alan", "B_nemesis": "Ghosts"}, {"A_age": 28,"A_name": "Alan", "B_character": "Alan", "B_nemesis": "Zombies"}]</code>'');'
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/hash-join.english.md]: This one.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/jaro-distance.english.md [hunk 10]
```diff
@@ -39,9 +39,9 @@ Write a function a that takes two strings as parameters and returns the associat
 ```yml
 tests:
   - text: <code>jaro</code> should be a function.
-    testString: assert(typeof jaro=='function','<code>jaro</code> should be a function.');
+    testString: assert(typeof jaro=='function');
   - text: <code>jaro("MARTHA", "MARHTA")</code> should return a number.
-    testString: assert(typeof jaro('MARTHA', 'MARHTA')=='number','<code>jaro()</code> should return a number.');
+    testString: assert(typeof jaro('MARTHA', 'MARHTA')=='number');
   - text: <code>jaro("MARTHA", "MARHTA")</code> should return <code>0.9444444444444445</code>.
     testString: assert.equal(jaro('MARTHA', 'MARHTA'), 0.9444444444444445,'<code>jaro("MARTHA", "MARHTA")</code> should return <code>0.9444444444444445</code>.');
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/jaro-distance.english.md]: 5 arguments in this file.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/s-expressions.english.md [hunk 11]
```diff
@@ -32,7 +32,7 @@ and turn it into a native data structure. (See the <a href="https://rosettacode.
 ```yml
 tests:
   - text: <code>parseSexpr</code> is a function.
-    testString: assert(typeof parseSexpr === 'function', '<code>parseSexpr</code> is a function.');
+    testString: assert(typeof parseSexpr === 'function');
   - text: <code>parseSexpr('(data1 data2 data3)')</code> should return <code>['data1', 'data2', 'data3']</code>
     testString: assert.deepEqual(parseSexpr(simpleSExpr), simpleSolution, "<code>parseSexpr('(data1 data2 data3)')</code> should return ['data1', 'data2', 'data3']");
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/s-expressions.english.md]: 2 arguments here.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sedols.english.md [hunk 12]
```diff
@@ -35,7 +35,7 @@ Check that each input is correctly formed, especially with respect to valid char
 ```yml
 tests:
   - text: <code>sedol</code> is a function.
-    testString: assert(typeof sedol === 'function', '<code>sedol</code> is a function.');
+    testString: assert(typeof sedol === 'function');
   - text: <code>sedol('a')</code> should return null.
     testString: assert(sedol('a') === null, "<code>sedol('a')</code> should return null.");
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sedols.english.md]: 4 arguments in this file.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-an-array-of-composite-structures.english.md [hunk 13]
```diff
@@ -20,7 +20,7 @@ Write a function that takes an array of objects as a parameter. The function sho
 ```yml
 tests:
   - text: <code>sortByKey</code> should be a function.
-    testString: assert(typeof sortByKey == 'function', '<code>sortByKey</code> should be a function.');
+    testString: assert(typeof sortByKey == 'function');
   - text: '<code>sortByKey([{key: 3, value: "foo"}, {key: 2, value: "bar"}, {key: 4, value: "baz"}, {key: 1, value: 42}, {key: 5, value: "another string"}])</code> should return an array.'
     testString: assert(Array.isArray(sortByKey([{key:3, value:"foo"}, {key:2, value:"bar"}, {key:4, value:"baz"}, {key:1, value:42}, {key:5, value:"another string"}])), '<code>sortByKey([{key:3, value:"foo"}, {key:2, value:"bar"}, {key:4, value:"baz"}, {key:1, value:42}, {key:5, value:"another string"}])</code> should return an array.');
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-an-array-of-composite-structures.english.md]: Fix this file.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/tokenize-a-string-with-escaping.english.md [hunk 14]
```diff
@@ -45,9 +45,9 @@ and using <code>|</code> as a separator and <code>^</code> as escape character,
 ```yml
 tests:
   - text: <code>tokenize</code> is a function.
-    testString: assert(typeof tokenize === 'function', '<code>tokenize</code> is a function.');
+    testString: assert(typeof tokenize === 'function');
   - text: <code>tokenize</code> should return an array.
-    testString: assert(typeof tokenize('a', 'b', 'c') === 'object', '<code>tokenize</code> should return an array.');
+    testString: assert(typeof tokenize('a', 'b', 'c') === 'object');
   - text: <code>tokenize('one^|uno||three^^^^|four^^^|^cuatro|', '|', '^') </code> should return <code>['one|uno', '', 'three^^', 'four^|cuatro', '']</code>
     testString: assert.deepEqual(tokenize(testStr1, '|', '^'), res1, "<code>tokenize('one^|uno||three^^^^|four^^^|^cuatro|', '|', '^') </code> should return ['one|uno', '', 'three^^', 'four^|cuatro', '']");
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/tokenize-a-string-with-escaping.english.md]: 2 arguments.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/top-rank-per-group.english.md [hunk 15]
```diff
@@ -52,17 +52,17 @@ one could rank the top-rated movie in each genre by calling
...
-    testString: assert.equal(res1[0][1].salary, 21900, 'First department must be D050');
+    testString: assert.equal(res1[0][1].salary, 21900);
   - text: The last department must be D202
-    testString: assert.equal(res1[3][3].dept, 'D202', 'The last department must be D202');
+    testString: assert.equal(res1[3][3].dept, 'D202');
   - text: <code>topRankPerGroup(1, ...)</code> must return only top ranking result per group.
-    testString: assert.equal(res2[2].length, 1, '<code>topRankPerGroup(1, ...)</code> must return only top ranking result per group.');
+    testString: assert.equal(res2[2].length, 1);
   - text: <code>topRankPerGroup(1, ...)</code> must return only top ranking result per group.
     testString: assert.equal(res3[2][1].name, 'Maze Runner', '<code>topRankPerGroup(1, ...)</code> must return only top ranking result per group.');
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/top-rank-per-group.english.md]: This one.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/towers-of-hanoi.english.md [hunk 16]
```diff
@@ -23,9 +23,9 @@ For example, the array <code>[['A', 'C'], ['B', 'A']]</code> indicates that the
 ```yml
 tests:
   - text: <code>towerOfHanoi</code> is a function.
-    testString: assert(typeof towerOfHanoi === 'function', '<code>towerOfHanoi</code> is a function.');
+    testString: assert(typeof towerOfHanoi === 'function');
   - text: <code>towerOfHanoi(3, ...)</code> should return 7 moves.
-    testString: assert(res3.length === 7, '<code>towerOfHanoi(3, ...)</code> should return 7 moves.');
+    testString: assert(res3.length === 7);
   - text: <code>towerOfHanoi(3, 'A', 'B', 'C')</code> should return <code>[['A','B'], ['A','C'], ['B','C'], ['A','B'], ['C','A'], ['C','B'], ['A','B']]</code>.
     testString: assert.deepEqual(towerOfHanoi(3, 'A', 'B', 'C'), res3Moves, "<code>towerOfHanoi(3, 'A', 'B', 'C')</code> should return [['A','B'], ['A','C'], ['B','C'], ['A','B'], ['C','A'], ['C','B'], ['A','B']].");
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/towers-of-hanoi.english.md]: 3 arguments here.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/generate-lower-case-ascii-alphabet.english.md [hunk 17]
```diff
@@ -20,7 +20,7 @@ Write a function to generate an array of lower case ASCII characters for a given
 ```yml
 tests:
   - text: <code>lascii</code> should be a function.
-    testString: assert(typeof lascii=='function','<code>lascii</code> should be a function.');
+    testString: assert(typeof lascii=='function');
   - text: <code>lascii("a","d")</code> should return an array.
     testString: assert(Array.isArray(lascii('a','d')),'<code>lascii("a","d")</code> should return an array.');
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/generate-lower-case-ascii-alphabet.english.md]: Fix this file.

**Review by vkWeb — CHANGES_REQUESTED** [review]
@RandellDawson Please remove the arguments mentioned in the files.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/align-columns.english.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/align-columns.english.md]: Fixed in separate commit.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/deal-cards-for-freecell.english.md (hunk 2 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/deal-cards-for-freecell.english.md]: Fixed in separate commit.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/deepcopy.english.md (hunk 3 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/deepcopy.english.md]: Fixed in separate commit.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/factorial.english.md (hunk 4 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/factorial.english.md]: Fixed in separate commit.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/fibonacci-sequence.english.md (hunk 5 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/fibonacci-sequence.english.md]: Fixed in separate commit.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-an-array-of-composite-structures.english.md (hunk 13 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-an-array-of-composite-structures.english.md]: Fixed in separate commit.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/tokenize-a-string-with-escaping.english.md (hunk 14 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/tokenize-a-string-with-escaping.english.md]: Fixed in separate commit.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/top-rank-per-group.english.md (hunk 15 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/top-rank-per-group.english.md]: Fixed in separate commit.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/towers-of-hanoi.english.md (hunk 16 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/towers-of-hanoi.english.md]: Fixed in separate commit.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/generate-lower-case-ascii-alphabet.english.md (hunk 17 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/generate-lower-case-ascii-alphabet.english.md]: Fixed in separate commit.

**Comment by RandellDawson** [comment]
@vkWeb Many of these were already fixed on "additional" PRs which addressed the ones missed in this PR, but I went ahead and fixed them anyway, just to make sure.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-an-array-of-composite-structures.english.md [hunk 18]
```diff
@@ -20,15 +20,15 @@ Write a function that takes an array of objects as a parameter. The function sho
...
+    testString: assert.deepEqual(sortByKey([{key:3, value:"foo"}, {key:2, value:"bar"}, {key:4, value:"baz"}, {key:1, value:42}, {key:5, value:"another string"}]), [{key:1, value:42}, {key:2, value:"bar"}, {key:3, value:"foo"}, {key:4, value:"baz"}, {key:5, value:"another string"}]);
   - text: '<code>sortByKey([{key: 3, name: "Joe"}, {key: 4, name: "Bill"}, {key: 20, name: "Alice"}, {key: 5, name: "Harry"}])</code> should return <code>[{key: 3, name: "Joe"}, {key: 4, name: "Bill"}, {key: 5, name: "Harry"}, {key: 20, name: "Alice"}]</code>.'
     testString: assert.deepEqual(sortByKey([{key:3, name:"Joe"}, {key:4, name:"Bill"}, {key:20, name:"Alice"}, {key:5, name:"Harry"}]), [{key:3, name:"Joe"}, {key:4, name:"Bill"}, {key:5, name:"Harry"}, {key:20, name:"Alice"}], '<code>sortByKey([{key:3, name:"Joe"}, {key:4, name:"Bill"}, {key:20, name:"Alice"}, {key:5, name:"Harry"}])</code> should return <code>[{key:3, name:"Joe"}, {key:4, name:"Bill"}, {key:5, name:"Harry"}, {key:20, name:"Alice"}]</code>.');
```
Comment (vkWeb) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-an-array-of-composite-structures.english.md]: This one here.

**Review by vkWeb — CHANGES_REQUESTED** [review]
@RandellDawson There's just one message argument left to be removed else everything looks fine to me.

**Comment by vkWeb** [comment]
Don't say thanks @RandellDawson 😅 I love helping freeCodeCamp :)

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-an-array-of-composite-structures.english.md (hunk 18 above)
Comment (ojeytonwilliams) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-an-array-of-composite-structures.english.md]: Good catch, @vkWeb, I'll add that while I fix the build error.  Shouldn't be long!

**Comment by ojeytonwilliams** [comment]
@vkWeb Sorry, it's not giving you credit as a co-author... let me try again.

**Comment by vkWeb** [comment]
@RandellDawson It's absolutely fine. I don't want any kind of credit, just merge this ;)

**Review by ojeytonwilliams — APPROVED** [review]
LGTM!
